README.md

```diff
@@ -68,6 +68,28 @@ $ nethelp  --cloud vdc --dc na
 [✓] http://ondemand.saucelabs.com:80 is reachable 200 OK
 ```
 
+### Idle server
+
```

Comment:
Can you add a *For Development* only note here.  Something to signal to the reader that they shouldn't/won't be accessing this as they'll need to install golang locally.

Reply:
@rgauthier-sauce See above^

Reply:
I updated the README.